core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -609,7 +611,7 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system) with FlatSpe
     }
 
     assert {
-      rqComplete.errorReport.message.contains("Invalid outputs: three_step.cgrep.count")
+      rqComplete.errorReport.message.contains("Invalid outputs: cgrep.count")
     }
```

Comment:
Just curious -- why the changes for three_step -> gcrep ?

Reply:
changed it back - it was originally because cromwell was returning inputs and outputs without the workflow name appended